Lib/test/test_importlib/source/test_file_loader.py

```diff
@@ -193,6 +193,9 @@ def test_file_from_empty_string_dir(self):
                 shutil.rmtree(pycache)
 
     @util.writes_bytecode_files
+    @unittest.skipIf(sys.platform == "os400",
+                     "On IBM i IFS doesn't support times past 2038"
+                     "os.utime of (2 ** 33 - 5) creates date in 2242")
```

Comment:
Perhaps it would be better to adjust the logic below that handles OSError and checks for EOVERFLOW. We could add a check for EINVAL (?) on IBM i instead.

Reply:
Fixed in 052edb5